src/05-derivatives/03-imaging.md

```diff
@@ -5,14 +5,119 @@ extent and resolution.
 
 ## Preprocessed, coregistered and/or resampled volumes
 
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        <datatype>/
+            <source_keywords>[_space-<space>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<ext>
+```
+
+
 Volumetric preprocessing does not modify the number of dimensions, and so
 the specifications in [Preprocessed or cleaned data][common_preproc]
 apply.
-In addition, all processed files include the following metadata JSON fields:
+The use of surface meshes and volumetric measures sampled to those meshes is
+sufficiently similar in practice to treat them equivalently.
+
+When two or more instances of a given derivative are provided with resolution
+(or surface sampling density) being the only difference between them, then the
+`res` (for *resolution* of regularly sampled N-D data) and/or `den` (for
+*density* of non-parametric surfaces) SHOULD be used to avoid name conflicts.
+Note that only files combining both regularly sampled (e.g., gridded) and surface
+sampled data (and their downstream derivatives) are allowed to present both `res`
+and `den` keywords simultaneously.
+
+!!! tip "Recommendation"
+    Although the `res` entity accepts any BIDS-valid `<label>`, it is
+    recommended to use one-based, consecutive indices as label to avoid
+    inconsistencies between the value of `res` and the metadata embedded
+    within the actual file.
```

Comment:
While on 02-common-data-types, there was this discussion: https://github.com/bids-standard/bids-specification/pull/301#discussion_r319862076

I'm not sure that this still applies, as the example is now `hi`/`low`, not one-based... I think we can probably just drop this tip.

Reply:
I feel this recommendation is still relevant, but not as relevant as to think it through to get a strong argument in favor - so I'm fine either way.

Reply:
Removed it for now. If it seems advisable, we can re-add. It just doesn't seem to match the surrounding text.